<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 10, 10);
</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1918</width>
       <height>998</height>
      </rect>
     </property>
     <widget class="QWidget" name="headerWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>300</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;</string>
      </property>
      <widget class="QFrame" name="header">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1920</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-bottom-color: rgb(75, 75, 75);
background-color: rgb(0, 0, 0);
border-style: outset;
border-width: 3px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="main_info_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 24pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
border: none;</string>
        </property>
        <property name="text">
         <string> Gaello X </string>
        </property>
       </widget>
       <widget class="QPushButton" name="notifications">
        <property name="geometry">
         <rect>
          <x>1550</x>
          <y>30</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;
border-width: 1px;
border-radius:24px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/alarm-bell.png</normaloff>../icons/alarm-bell.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="help">
        <property name="geometry">
         <rect>
          <x>1640</x>
          <y>30</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;
border-width: 1px;
border-radius:24px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/help-web-button.png</normaloff>../icons/help-web-button.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="menu">
        <property name="geometry">
         <rect>
          <x>1710</x>
          <y>30</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;
border-width: 1px;
border-radius:24px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/round (1).png</normaloff>../icons/round (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="docs">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>50</y>
          <width>100</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
font: 10pt &quot;Candara Light&quot;;
border-bottom-color: rgb(75, 75, 75);
color: rgb(150, 150, 150);
border-style: outset;
border-width: 1px;</string>
        </property>
        <property name="text">
         <string>Docs</string>
        </property>
       </widget>
       <widget class="QPushButton" name="demo">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>50</y>
          <width>100</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
font: 10pt &quot;Candara Light&quot;;
border-bottom-color: rgb(75, 75, 75);
color: rgb(150, 150, 150);
border-style: outset;
border-width: 1px;</string>
        </property>
        <property name="text">
         <string>Demo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="aboutUs">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>50</y>
          <width>100</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
font: 10pt &quot;Candara Light&quot;;
border-bottom-color: rgb(75, 75, 75);
color: rgb(150, 150, 150);
border-style: outset;
border-width: 1px;</string>
        </property>
        <property name="text">
         <string>Abouts us</string>
        </property>
       </widget>
       <widget class="QPushButton" name="account">
        <property name="geometry">
         <rect>
          <x>1470</x>
          <y>30</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;
border-width: 1px;
border-radius:24px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/profile-user.png</normaloff>../icons/profile-user.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="main_info">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>72</width>
          <height>72</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
border: none</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../logo/cube-logo-big.png</normaloff>../logo/cube-logo-big.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="closeApp">
        <property name="geometry">
         <rect>
          <x>1860</x>
          <y>30</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;
border-width: 1px;
border-radius:24px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/remove-button.png</normaloff>../icons/remove-button.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="minimize">
        <property name="geometry">
         <rect>
          <x>1790</x>
          <y>30</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;
border-width: 1px;
border-radius:24px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/minimize.png</normaloff>../icons/minimize.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="bulkWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1920</width>
        <height>840</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
border-style: none;
border-width: 0px;
</string>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>10</y>
         <width>500</width>
         <height>800</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-radius: 12px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>480</width>
          <height>780</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(50, 50, 50);
border-style: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="marketButton">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>390</y>
           <width>60</width>
           <height>60</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/next (1).png</normaloff>../icons/next (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="exploreForexTitle">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>460</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Explore forex and cryptos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="exploreForexLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>460</width>
           <height>300</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: outset;
color: rgb(200, 200, 200);
border-width: 2px;
border-radius: 12px;
border-color: rgb(150, 150, 150);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Explore Market on the forex and cryptos side.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;We provide every details you would like to know.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;These information are refreshed on a daily basis, allowing you to get fresh and up-to-date market data.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="marketGif">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>470</y>
           <width>460</width>
           <height>300</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>1400</x>
         <y>10</y>
         <width>500</width>
         <height>800</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-radius: 12px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>480</width>
          <height>780</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(50, 50, 50);
border-style: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="exploreStocksTitle">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>460</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Focus on stock&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="exploreStocksLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>370</y>
           <width>460</width>
           <height>300</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: outset;
color: rgb(200, 200, 200);
border-width: 2px;
border-radius: 12px;
border-color: rgb(150, 150, 150);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;If you would like to focus on a particular stock,&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;You will find here everything you need to know in a single search&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;(Financial statements, key metrics, ratios, etc.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="assetButton">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>690</y>
           <width>60</width>
           <height>60</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 30px;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/next (1).png</normaloff>../icons/next (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="stockGif">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>460</width>
           <height>300</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QFrame" name="chat">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>600</width>
         <height>150</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">;
background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-color: rgb(129, 0, 97);
border-style: solid;
border-width: 2px;
border-radius: 12px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="chat_h1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>481</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Centry Gothic&quot;;
color: rgb(250, 250, 250);
border-style: none;
background: none;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chat with Janine&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="chat_p">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>581</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
border-style: none;
background: none;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Janine is the AI model that empowers you to make better investment decisions by backing you up with loads of real-time data.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>-3</number>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="explore">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>660</y>
         <width>600</width>
         <height>150</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">;
background-color: qlineargradient(spread:pad, x1:0.886772, y1:0.863182, x2:0.034, y2:0.0567727, stop:0 rgba(50, 50, 50, 100), stop:1 rgba(0, 0, 0, 100));
border-color: rgb(0, 96, 144);
border-style: solid;
border-width: 2px;
border-radius: 12px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="explore_p">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>581</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
border-style: none;
background: none;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;An outlook of the market and a some of the news we use to decrypt it. The data is updated on a daily basis.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>-3</number>
        </property>
       </widget>
       <widget class="QLabel" name="explore_h1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>481</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Centry Gothic&quot;;
color: rgb(250, 250, 250);
border-style: none;
background: none;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Explore Focus feed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="pitch">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>541</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Centry Gothic&quot;;
color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;A Large language Model for future events prediction.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="descr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>541</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Centry Gothic&quot;;
color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Shape your next investment or trading decision with the analytical power of an AI.&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../rsrc/images/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
